frontend/src/features/projects/spl/steps/__snapshots__/SplStep.test.tsx.snap

```diff
@@ -151,7 +151,7 @@ exports[`SPL Approval Step renders correctly 1`] = `
                       placeholder="--/--/----"
                       readOnly={false}
                       type="text"
-                      value="07/14/2020"
+                      value="07/15/2020"
```

Comment:
lol, still have this bug showing up in our tests.  Hope someone can resolve this soon.

Reply:
the solution is to delete this snapshot test (for now) - painful to have to update this daily